pulsar-broker/src/main/java/org/apache/pulsar/broker/service/Consumer.java

```diff
@@ -140,11 +142,13 @@ public class Consumer {
 
     private long negtiveUnackedMsgsTimestamp;
 
+    private SchemaData schemaData;
```

Comment:
good way. but maybe we only use the SchemaType is enough

Reply:
`Schemadata` contains `SchemaType`. I think `Schemadata` is better. HAHA

Reply:
If there are other needs we can use SchemaData? it seems that only `SchemaType` is enough for now

Reply:
OK